Werewolf for Telegram/Werewolf Node/Werewolf.cs

```diff
@@ -761,8 +761,11 @@ public void HandleReply(CallbackQuery query)
 
                 if (player.CurrentQuestion.QType == QuestionType.Lynch)
                 {
-                    var msg = GetLocaleString("PlayerVotedLynch", player.GetName(), target.GetName());
-                    SendWithQueue(msg);
+                    if (DbGroup.EnableSecretLynch == false)
+                    {
+                        var msg = GetLocaleString("PlayerVotedLynch", player.GetName(), target.GetName());
```

Comment:
This needs to be fixed.  EnableSecretLynch can be null, which will be the default for all the groups that already exist.  should be `if (DbGroup.EnableSecretLynch != true)`